packages/components/stories/icon/icons.js

```diff
@@ -645,6 +646,9 @@ export const icons =
         'IcContractStrike',
         'IcContractTarget',
     ],
+    'ctrader': [
```

Comment:
I do not think if we need to include icons file changes in this PR

Reply:
I think this file changes are irrelevant and can be reverted back
@shontzu-deriv